app/components/pair-compare.jsx

```diff
@@ -142,11 +143,13 @@ function PairCompare(props) {
     <div className={classes.container} {...otherProps}>
       <div className={classes.mainPointContainer}>
         <div className={classes.mainPointSubject}>{mainPoint.subject}</div>
+        <div style={{ marginBottom: '1rem'}}></div>
         <div className={classes.mainPointDescription}>{mainPoint.description}</div>
+        <div style={{ marginBottom: '2rem'}}></div>
       </div>
 
       <span className={isSelectionComplete ? classes.statusBadgeComplete : classes.statusBadge}>{`${pointsIdxCounter <= whyRankList.length ? pointsIdxCounter : whyRankList.length} out of ${whyRankList.length}`}</span>
-
+      <div style={{ marginBottom: '2.5rem' /*increase margin at the bottom of n out of 6 label*/ }}></div>
```

Comment:
can this be accomplishd by adjusting the margin of the lowerContainer class instad of adding another div?